### DISCUSSION TITLE
Inconsistent documentation on either 'django_celery_beat.schedulers:DatabaseScheduler' or just 'django'

### DISCUSSION THREAD

**Discussion by mfonville** [celery/django-celery-beat#475] [Ideas]
### Summary:

* n the Celery configuration documentation: https://docs.celeryproject.org/en/stable/userguide/periodic-tasks.html#beat-custom-schedulers it states to use `--scheduler django_celery_beat.schedulers:DatabaseScheduler`
* In the Celery CLI documentation it states that `-S` and `--scheduler` are fully equivalent: https://docs.celeryproject.org/en/stable/reference/cli.html#cmdoption-celery-beat-S
* In the celery beat documentation Readme https://github.com/celery/django-celery-beat/blame/master/README.rst#L212 it states first `--scheduler django_celery_beat.schedulers:DatabaseScheduler` 
* In the celery beat documentation Readme https://github.com/celery/django-celery-beat/blame/master/README.rst#L217 it also states that `-S django` should work

It is not clear from the configuration and readme if `-S` and `--scheduler` are truly equivalent.
Neither it is clear `django_celery_beat.schedulers:DatabaseScheduler` or `django` are equivalent and if any of them is the preferred method.

My suggestion is to explicitly stick to either only using `-S` or using only `--scheduler` in all the documented examples if they are fully equivalent.
For the scheduler itself, if they are somehow different in how/why they work, describe why, and give it as 2 separate examples.

**Comment by auvipy** [comment]
can you dig a little bit more of it?
